docker-entrypoint.sh

```diff
@@ -19,6 +19,13 @@ run_as() {
     fi
 }
 
+if [ -n "${NEXTCLOUD_UID+x}" ]; then
+    usermod -u "$NEXTCLOUD_UID" www-data
+fi
+if [ -n "${NEXTCLOUD_GID+x}" ]; then
+    groupmod -g "$NEXTCLOUD_GID" www-data
```

Comment:
group is always set to root in https://github.com/nextcloud/docker/blob/35fd42d0429ed0deaf2081f8210f31ce3758ab40/docker-entrypoint.sh#L40